components/devtools/network_handler.rs

```diff
@@ -26,17 +26,19 @@ pub(crate) fn handle_network_event(
     netevent_actor_name: String,
     mut connections: Vec<TcpStream>,
     network_event: NetworkEvent,
-    watcher_name: String,
 ) {
     let mut actors = actors.lock().unwrap();
+    let watcher_name = {
+        let actor_ref = actors.find::<NetworkEventActor>(&netevent_actor_name);
+        actor_ref.watcher_name.clone()
+    };
+    let actor = actors.find_mut::<NetworkEventActor>(&netevent_actor_name);
```

Comment:
Can we store a clone of the watcher name here instead of looking up NetworkEventActor two times?